uniflow/flow/config.py

```diff
@@ -540,6 +539,57 @@ class RaterForClassificationBedrockClaudeConfig(RaterConfig):
     )
 
 
+@dataclass
+class RaterForClassificationHuggingfaceConfig(RaterConfig):
+    """
+    The configuration primarily focuses on setting up the parameters for utilizing Huggingface model to
+    evaluate the correctness of answers in relation to given questions and contexts.
+
+    Attributes:
+        flow_name (str): Name of the rating flow, default is "RaterFlow".
+        model_config (ModelConfig): Configuration for the huggingeface model. Includes model_name("mistralai/Mistral-7B-Instruct-v0"),
+                                    model_server ("HuggingfaceModelServer"), batch_size (1), neuron (False),
+                                    load_in_4bit (False), load_in_8bit (True)
+        label2score (Dict[str, float]): Mapping of labels to scores, default is {"Yes": 1.0, "No": 0.0}.
+        guided_prompt_template (GuidedPrompt): Template for guided prompts used in rating. Includes instructions
+                                               for rating, along with examples that detail the context, question,
+                                               answer, label, and explanation for each case.
+    """
+
+    model_config: ModelConfig = field(
+        default_factory=HuggingfaceModelConfig(
+            response_start_key="explanation",
+            response_format={"type": "text"},
+            batch_size=1
```

Comment:
I think you miss a comma here which trigger a black failure during build time. If you have not do so, follow https://www.notion.so/goldpiggy/Python-Linter-and-formatter-Setup-30fb3b81f0904af889832e4c697c5ec9?pvs=4 to properly setup your dev space.

Reply:
sorry I forget to press command+s before push